packages/styles/skip-link.css

```diff
@@ -27,10 +25,13 @@
 }
 
 .SkipLink a {
+  min-width: 208px;
```

Comment:
Nitpick: If we want to enforce a min-width, we should probably use rems here.

`208px` === `13rem`